paddle/fluid/operators/detail/send_recv.proto

```diff
@@ -14,6 +14,8 @@ limitations under the License. */
 syntax = "proto3";
 package sendrecv;
 
+// option cc_generic_services = true;
```

Comment:
Can delete this if not needed? Curious what will  this line affect?

Reply:
https://developers.google.com/protocol-buffers/docs/proto

```
Whether or not the protocol buffer compiler should generate abstract service code based on services definitions in C++, Java, and Python, respectively. For legacy reasons, these default to true. However, as of version 2.3.0 (January 2010), it is considered preferrable for RPC implementations to provide code generator plugins to generate code more specific to each system, rather than rely on the "abstract" services
```
Grpc uses the code generator plugin to generate  more specific to each system, and it needs this to be false. While brpc uses the default generated abstract service code, and it needs this to be true. I will fix this to be compatbile in next PR.

Reply:
Thanks for this information!